EditImprovements/README.md

```diff
@@ -0,0 +1,5 @@
+Description
+-----------
+
+Edit.Locate and ~.Search sometimes position the viewer that the target location is outside.
+This patch insures that the target location will always be at the first line of the viewer.
```

Comment:
```suggestion
EditImprovements - Edit.Locate and ~.Search display the target location in the first line of the viewer

Description
-----------

When invoking them on small viewers, Edit.Locate and ~.Search sometimes position the text in the viewer that the target location is beyond the end of the viewer.
This patch insures that the target location will always be at the first line of the viewer.

Installation
------------

- Apply [`Edit.patch`](Edit.patch) to `Edit.Mod`.

- Recompile `Edit.Mod`:

      ORP.Compile Edit.Mod ~
```

Suggested edits above.

Reply:
Also, please use the "Resolve conflicts" button below to resolve the merge conflict in the readme file. Or if you prefer, rebase your branch on top of my current version.

Reply:
Yup, I noticed that, but needed more time to find the solution with svn: merge -c, fetch changes from master on GitHub, update local copy, change README.md and commit.

Reply:
Good, I was unsure since I got about a dozen email notifications for this pull request, including one that you closed and reopened it. (In case you know you need more time and prefer feedback later, there is a "Convert to draft" button in the upper right corner. So GitHub will not notify me until you finished editing and remove the draft status).